### PR TITLE
UI: Added Additional Stats for BlockViewer

### LINKED ISSUE #3219: Block Viewer: Show some additional stats
We can show some more information in the Block Viewer:
- Last refresh time: it's already available in the API response
- Total number of blocks (across all sources)
- Total number of blocks in each group
- Number of overlapping blocks (in each row/group and overall)
- Which blocks are currently being compacted/downsampled (makes sense only when the Block Viewer is run from compactor)

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <adikrish@redhat.com>

* [x] I added CHANGELOG entry for this change.

## Changes

This PR Fixes #3219 

- Added number of blocks
- Last refresh
- Added number of blocks in each group

### CODE REVIEW

File: pkg/ui/react-app/src/thanos/pages/blocks/blocks.module.css [hunk 1]
```diff
@@ -189,14 +215,4 @@
...
-.blockInput {
-  margin-bottom: 12px;
-}
-
-.blockFilter {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-}
+}
```
Comment (yeya24) [inline pkg/ui/react-app/src/thanos/pages/blocks/blocks.module.css:218]: Please add a new line here

**Review by yeya24 — COMMENTED** [review]
@adithyaakrishna Can you please add a screenshot of this change?

File: pkg/ui/react-app/src/thanos/pages/blocks/Blocks.tsx [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
+import { useQueryParams, withDefault, NumberParam } from 'use-query-params';
 import { withStatusIndicator } from '../../../components/withStatusIndicator';
 import { useFetch } from '../../../hooks/useFetch';
 import PathPrefixProps from '../../../types/PathPrefixProps';
 import { Block } from './block';
 import { SourceView } from './SourceView';
 import { BlockDetails } from './BlockDetails';
-import { BlockSearchInput } from './BlockSearchInput';
-import { BlockFilterCompaction } from './BlockFilterCompaction';
-import { sortBlocks, getBlockByUlid, getFilteredBlockPools } from './helpers';
```
Comment (yeya24) [inline pkg/ui/react-app/src/thanos/pages/blocks/Blocks.tsx:13]: Why do we need to remove those lines?

**Review by matej-g — COMMENTED** [review]
I think this is looking good so far! Do you think we could get a screenshot as @yeya24 said to see how it looks? :slightly_smiling_face:

**Pull request closed without merging**